### PR TITLE
Implement adding and delete people to board

### PR DESCRIPTION
**Pull request by dwantula**

### CODE REVIEW

File: src/shared/components/Input/Input.jsx [hunk 1]
```diff
@@ -1,31 +1,55 @@
...
   onChange: PropTypes.func,
-  name: PropTypes.string,
+  onBlur: PropTypes.func,
+  value: PropTypes.string,
   type: PropTypes.string,
+  inputRef: PropTypes.objectOf(
+    PropTypes.shape({
+      id: PropTypes.string.isRequired,
+      name: PropTypes.string.isRequired,
+    }),
```
Comment (mi-chal) [inline src/shared/components/Input/Input.jsx]: I don't think this correct the structure for ref.

File: src/shared/components/PeopleList/PeopleList.jsx [hunk 2]
```diff
@@ -11,19 +11,21 @@ function PeopleList() {
...
     <div className="people-card">
-      {people.map(({ id, name }) => (
-        <PersonCard key={id} name={name} />
-      ))}
       <Button
         onClick={addNewPerson}
         className="people-card__button-add-person"
         text="Add Person"
       />
+      <div style={{ display: 'flex', flexWrap: 'wrap' }}>
```
Comment (mi-chal) [inline src/shared/components/PeopleList/PeopleList.jsx]: Do not use inline styles.

File: src/shared/components/PersonCard/PersonCard.jsx [hunk 3]
```diff
@@ -1,66 +1,76 @@
...
+          className="person-card__input"
+        />
+      </div>
+      <div
+        style={{
+          display: 'flex',
+          marginTop: 'auto',
+          padding: '20px 10px',
+          justifyContent: 'space-between',
+        }}
```
Comment (mi-chal) [inline src/shared/components/PersonCard/PersonCard.jsx]: Do not use inline style

File: src/shared/components/PeopleList/PeopleList.jsx (hunk 2 above)
Comment (dwantula) [reply src/shared/components/PeopleList/PeopleList.jsx]: Fixed

File: src/shared/components/PersonCard/PersonCard.jsx (hunk 3 above)
Comment (dwantula) [reply src/shared/components/PersonCard/PersonCard.jsx]: Fixed

File: src/shared/components/Input/Input.jsx (hunk 1 above)
Comment (dwantula) [reply src/shared/components/Input/Input.jsx]: Fixed

File: src/shared/components/Input/Input.jsx [hunk 4]
```diff
@@ -1,31 +1,54 @@
...
-  name: PropTypes.string,
+  onBlur: PropTypes.func,
+  value: PropTypes.string,
   type: PropTypes.string,
+  // inputRef: PropTypes.string,
+  // inputRef: PropTypes.objectOf(
+  //   PropTypes.shape({
+  //     name: PropTypes.string,
+  //   }),
+  // ),
```
Comment (mi-chal) [inline src/shared/components/Input/Input.jsx]: Remove comments

File: src/shared/components/Input/Input.jsx [hunk 5]
```diff
@@ -1,31 +1,54 @@
...
+  inputRef: PropTypes.oneOfType([
+    PropTypes.func,
+    PropTypes.shape({ current: PropTypes.instanceOf(Element) }),
```
Comment (mi-chal) [inline src/shared/components/Input/Input.jsx]: `PropTypes.instanceOf(HTMLInputElement)`

Comment (dwantula) [reply src/shared/components/Input/Input.jsx]: Fixed

File: src/shared/components/Input/Input.jsx (hunk 4 above)
Comment (dwantula) [reply src/shared/components/Input/Input.jsx]: Fixed

**Review by mi-chal — APPROVED** [review]

**Pull request merged**